test/e2e/adminUI/tests/group002Home/uxTestHomeView.js

```diff
@@ -102,11 +102,11 @@ module.exports = {
 			.click('@nameListSubmenu')
 			.waitForListScreen();
 
-		browser.adminUIListScreen
-			.click('@singleItemDeleteIcon');
+		browser.adminUIListScreen.clickDeleteItemIcon([
+			{ row: 1, column: 1 }
```

Comment:
@jstockwin not sure and why make that assumption?  The list can be configured with options that cause other icons to show to the left of the items.  Not sure why you think this should be a special case.